scss/atoms/Meter/_Meter.scss

```diff
@@ -0,0 +1,58 @@
+// ==========================================================================
+// Meter
+// ==========================================================================
+meter {
+	-webkit-appearance: none;
+	width: 550px;
+	height: 25px;
+}
+
+.meter::-webkit-meter-bar, .multiColor::-webkit-meter-bar, .stripMeter::-webkit-meter-bar {
```

Comment:
Why is all your meter styling based off of class names?  This is supposed to be the styling for the base meter property.